src/main/java/com/hollingsworth/arsnouveau/common/items/DominionWand.java

```diff
@@ -169,7 +172,7 @@ public void appendHoverText(@NotNull ItemStack stack, @NotNull TooltipContext wo
         if (data.storedPos().isEmpty()) {
             tooltip.add(Component.translatable("ars_nouveau.dominion_wand.no_location"));
         } else {
-            tooltip.add(Component.translatable("ars_nouveau.dominion_wand.position_stored", getPosString(data.getValidPos())));
+            tooltip.add(Component.translatable("ars_nouveau.dominion_wand.position_stored", getGlobalPosString(data.getValidPos())));
```

Comment:
Should getPosString be removed/deprecated? Is it used anywhere else?

Reply:
Yep, they are still used in RedstoneRelayTile and RelayTile.